cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -293,21 +297,25 @@ def configure_ua(token, enable=None):
         util.logexc(LOG, msg)
         raise RuntimeError(msg) from e
 
-    enable_errors = []
+    enable_errors: List[Tuple[str, str]] = []
     for service in enable:
         try:
-            cmd = ["ua", "enable", "--assume-yes", service]
-            subp.subp(cmd, capture=True)
+            cmd = ["ua", "enable", "--assume-yes", "--format", "json", service]
+            enable_cmd_result = subp.subp(cmd, capture=True)
```

Comment:
I think we minimally have to have an rcs to allow for exit 1 here otherwise enable_cmd_result will not be set resulting in UnboundLocalErrors on enable_cmd_result if the first item in an enable: [esm-apps] is already enabled.
```suggestion
            enable_cmd_result = subp.subp(cmd, capture=True, rcs={0,1})
```
By providing for rcs={0,1} we may not actually need the except ProcessExecutionError handling, but I think we should leave it in in case UA adds more return codes from this CLI call besides 0 and 1. We should catch any errors which exit 1 in your following for loop anyway.

Reply:
Done, and unit-covered.